### PR TITLE
Add Mailchimp signup

### PR DESCRIPTION
**Pull request by j-hannah**
cc: @sirkitree Going to merge, but please let me know if you see anything of concern or note. 😄

### CODE REVIEW

File: src/components/Curated/Curated.js [hunk 1]
```diff
@@ -16,15 +17,30 @@ const styles = StyleSheet.create({
...
     {
-      props.articles.map((article, i) =>
-        <div key={i} className={css(styles.article)}>
-          <h2 className={css(global.title)}><a className={css(global.link, global.visited, global.hover)} href={article.fields.link}>{article.fields.title}</a></h2>
-          <div className={css(global.bodyText)} dangerouslySetInnerHTML={{ __html: marked(article.fields.body) }} />
-          <footer className={css(styles.footer)}>
-            <span>Source: {article.fields.source}</span>
-          </footer>
+      props.articles.map((article, i) => (
+        ((i + 1) % 11 === 0 && i < 78) ?
```
Comment (sirkitree) [inline src/components/Curated/Curated.js]: Does this mean you inject the mailchimp thing every 78 articles?

File: src/containers/HomePage/HomePage.js [hunk 2]
```diff
@@ -85,22 +74,20 @@ export class HomePage extends PureComponent {
...
-              <Curated articles={curatedArticles} />
-              : <p>No articles currently available</p>
-            }
-            {curatedArticles && (totalCuratedArticles > totalCuratedArticlesLoaded) &&
-              <div className={css(styles.buttonWrapper)}>
-                <Button text="Load More" onClick={this.loadMore} />
-              </div>
-            }
-          </div>
+          {curatedArticles ?
```
Comment (sirkitree) [inline src/containers/HomePage/HomePage.js]: Not sure I'll ever get used to this syntax, it's difficult to read for me.

File: src/components/Curated/Curated.js (hunk 1 above)
Comment (j-hannah) [inline src/components/Curated/Curated.js]: No, it gets injected every 11 and after appearing 7 times doesn't appear again.

File: src/containers/HomePage/HomePage.js (hunk 2 above)
Comment (j-hannah) [inline src/containers/HomePage/HomePage.js]: It took me a while as well. 😄

File: src/components/Curated/Curated.js (hunk 1 above)
Comment (j-hannah) [inline src/components/Curated/Curated.js]: Eh, I have this off by one on the first pass. I'll put together another PR for that fix. Thanks for looking at it.

**Pull request merged**